opennms-config/src/main/java/org/opennms/netmgt/filter/JdbcFilterDao.java

```diff
@@ -708,6 +713,7 @@ private String parseRule(final List<Table> tables, final String rule) throws Fil
             }
             regex.appendTail(tempStringBuff);
             sqlRule = tempStringBuff.toString();
+            if(!sqlRule.isEmpty())
```

Comment:
good practice is to use braces even for a single line, please take care of spaces also, no spotless here

Reply:
resolved added braces.